config/logcollector_manifest_full

```diff
@@ -0,0 +1,73 @@
+echo,### Probing Directories ###
```

Comment:
This file is identical to IID's [manifest file](https://github.com/Azure/azure-diskinspect-service/blob/master/pyServer/manifests/linux/diagnostic), collection option diagnostic.